MMR.Randomizer/Randomizer.cs

```diff
@@ -769,7 +771,7 @@ private bool CheckMatch(Item currentItem, Item target)
         private void PlaceItem(Item currentItem, List<Item> targets)
         {
             var currentItemObject = ItemList[currentItem];
-            if (currentItemObject.NewLocation.HasValue)
+            if (currentItemObject.NewLocation.HasValue || PlandoPlacedItems.Contains(currentItem))
```

Comment:
I might have misunderstood the problem but here is the issue and my attempted solution:

The ItemList is really the list of remaining checks that have to be filled. There is no list of items not yet placed for me to use (that I can find), and items are added statically by some if not all Place* function calls inRandomizer.cs::RandomizeItems(). The PlaceItem function does not check if an item has already been placed somewhere before, and without a list I don't know how to allow it to check for that. Without the list of previously placed plando items, plando starves MMR for checks, because every plando item gets placed twice (once by plando, once statically in the functions), and so every plando ends with an error abouts of "Unable to Place Item <Item> anywhere". Creating a list of previously placed items and checking that solves the issue.

I didn't want to rewrite all the place items functions so that they just use a list of items, because I don't like replacing that much code without know why it was written that way to begin with (was it just debugging, to help find an old issue?). If we had a list of items to place we could search for the item in the list and remove it during plando, which is a probably better than statically iterating over items.

Reply:
After you do `PlaceItem(item, new List<Item> { check });` if it successfully places it, it sets `ItemList[item].NewLocation = check;`. So later on if you try to place that same item again, it will already have the `.NewLocation` set.

Reply:
The issue would appear to be that the Items we pull from the json are not from ItemList, so changes to one or the other are isolated.

Need to rewrite so that the PlaceItem function gets objects from ItemList

Reply:
This section has been completely rewritten now.